### PR TITLE
COMMERCE-11289

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-11289

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase [hunk 1]
```diff
@@ -2440,6 +2440,74 @@ definition {
...
+				value1 = "MIN93015");
+		}
+
+		task ("Then results are shown") {
+			AssertElementPresent(
+				key_productSku = "MIN93015",
+				locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_SEARCH_RESULT");
+		}
+	}
+
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase:2510]: I would recommend adding another testcase in which we have products with only expired SKUs

**Comment by Brunno0liveira** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase [hunk 2]
```diff
@@ -2440,7 +2440,83 @@ definition {
 		}
 	}
 
-	@description = "This is a test for COMMERCE-11289. Quick add-to-cart works properly when one product is without an SKU"
+	@description = "This is a test for COMMERCE-11289. Quick add-to-cart works properly when one product is with SKUs expired"
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase:2443]: ...when a product has only expired SKUs"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase [hunk 3]
```diff
@@ -2440,7 +2440,83 @@ definition {
...
+	@priority = 5
+	test CanUseQuickAddToCartWhenOneProductHasExpiredSKU {
+		property portal.acceptance = "true";
+
+		task ("Given a product has a expired SKU") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase:2448]: Given a product has only expired SKUs

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase [hunk 4]
```diff
@@ -2440,7 +2440,83 @@ definition {
...
+				noOption = "true",
+				optionValuesList = "BLACK:10.0");
+
+			Click(
+				key_value = "BLACKSKU",
+				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+			MenuItem.click(menuItem = "Delete");
+
+			AssertElementPresent(
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase:2465]: I would also add an "AssertElementNotPresent" for the SKU tag "Approved"

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**